_pages/en_US/alternate-exploits.txt

```diff
@@ -4,7 +4,9 @@ title: "Alternate Exploits"
 
 {% include toc title="Table of Contents" %}
 
-MSET9 is currently the only exploit for 3DS / 3DS XL / 2DS consoles on firmware 11.17.0 that is free and does not require any software. If you're unable to use it, you'll need to use one of these methods, which all have some sort of prerequisite.
+This is a list of methods that work on all firmware but require extra hardware or software.
```

Comment:
firmware should be plural
If we are keeping this page on the guide then methods may not work on all firmwares